src/pages/api/models/task.ts

```diff
@@ -0,0 +1,19 @@
+export interface Task {
+  id: string;
+  text: string;
+  columnId: string;
+  position: number;
+}
+
+export const TaskSchema = {
```

Comment:
Move this file away from api folder. Everything inside that can be called by a client.